### ISSUE TITLE
Package is not available on apm anymore

### ISSUE THREAD

**Issue by BuonOmo** [johanlunds/atom-ruby-debugger#14]
$ apm install ruby-debugger
    Installing ruby-debugger to /Users/foo/.atom/packages ✗
    Request for package information failed: Not Found

And version is:

    $ apm -v
    apm  1.19.0
    npm  3.10.10
    node 6.9.5 x64
    atom 1.27.1
    python 2.7.15
    git 2.15.1

**Comment by johanlunds** [comment]
Yes, sorry I unregistered the package to allow someone else to make a
ruby-debugger.

I don't think my package works anymore.

**Comment by BuonOmo** [comment]
Ok, i'll try to install it on my own and see what happens then

**Comment by johanlunds** [comment]
Yes, go ahead! Good luck!
